monitoring-dashboard/distribution/src/main/resources/bin/yajsw/README.txt

```diff
@@ -0,0 +1,9 @@
+Running WSO2 Micro Integrator Dashboard as a Windows Service
+------------------------------------------------
+
+1. Download the latest stable version of YAJSW from the project home page. (http://yajsw.sourceforge.net/)
+2. Unzip the YAJSW archive and place the provided wrapper.conf file (this directory) inside <YAJSW.Home.Dir>/conf.
+3. Set 'java_home' and 'wso2mi_dashboard_home' system properties
+4. Start the product as a windows service. (batch scripts are found under <YAJSW.Home.Dir>/bat)
+
+For more detailed info please refer to Carbon wiki documentation (http://docs.wso2.org/wiki/dashboard.action) and YAJSW project documentation.
```

Comment:
shall we update http://docs.wso2.org/wiki/dashboard.action ?

Reply:
Removed this line since it is outdated